api/src/pcapi/routes/backoffice_v3/collective_offers.py

```diff
@@ -161,37 +161,88 @@ def get_validate_collective_offer_form(collective_offer_id: int) -> utils.Backof
 @list_collective_offers_blueprint.route("/<int:collective_offer_id>/validate", methods=["POST"])
 @utils.permission_required(perm_models.Permissions.FRAUD_ACTIONS)
 def validate_collective_offer(collective_offer_id: int) -> utils.BackofficeResponse:
-    collective_offer = educational_models.CollectiveOffer.query.get_or_404(collective_offer_id)
-
-    new_validation_status = OfferValidationStatus.APPROVED
-    if collective_offer.validation != OfferValidationStatus.PENDING:
-        flash("Seules les offres collectives en attente peuvent être validées", "warning")
-        return redirect(request.referrer or url_for("backoffice_v3_web.collective_offer.list_collective_offers"), 303)
-    collective_offer.validation = new_validation_status
-    collective_offer.isActive = True
-
-    try:
-        db.session.commit()
-    except Exception:  # pylint: disable=broad-except
-        flash("Une erreur est survenue lors de la validation de l'offre", "warning")
-        return redirect(request.referrer, 400)
-    search.async_index_collective_offer_ids([collective_offer.id])
-
-    flash("L'offre a bien été validée", "success")
-    collective_offer.lastValidationDate = datetime.datetime.utcnow()
-    collective_offer.lastValidationType = OfferValidationType.MANUAL
-    recipients = (
-        [collective_offer.venue.bookingEmail]
-        if collective_offer.venue.bookingEmail
-        else [recipient.user.email for recipient in collective_offer.venue.managingOfferer.UserOfferers]
+    _batch_validate_or_reject_collective_offers(OfferValidationStatus.APPROVED, [collective_offer_id])
+    return redirect(
+        request.referrer or url_for("backoffice_v3_web.collective_offer.list_collective_offers"),
+        303,
     )
```

Comment:
```suggestion
    return redirect(request.referrer or url_for("backoffice_v3_web.collective_offer.list_collective_offers"), 303)
```
pas besoin de changer